src/api.v2/routes/cellLevelMeta.js

```diff
@@ -0,0 +1,24 @@
+const {
+  uploadCellLevelMetadata, updateCellLevelMetadata, downloadCellLevelFile, deleteMetadata,
+} = require('../controllers/cellLevelMetaController');
+
+const { expressAuthorizationMiddleware } = require('../middlewares/authMiddlewares');
+
+module.exports = {
+  'cellLevelMeta#uploadCellLevelMetadata': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => uploadCellLevelMetadata(req, res).catch(next),
+  ],
+  'cellLevelMeta#updateCellLevelMetadata': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => updateCellLevelMetadata(req, res).catch(next),
+  ],
```

Comment:
is the `updateCellLevelMetadata`route used or was this necessary for the "replace" functionality, which we removed?

Reply:
used when the file is uploaded, to update its upload status.